lib/cn/kvset.c

```diff
@@ -2210,8 +2210,13 @@ kvset_iter_kblock_read(struct work_struct *rock)
     if (ev(err))
         goto done;
 
+    /* [HSE_REVISIT] kr->pc belongs to cn, this thread may be running after cn_close() and
+     * accessing kr->pc would cause a segfault. Uncomment after fixing this.
+     */
+#if 0
     perfc_inc(kr->pc, PERFC_RA_CNCOMP_RREQS);
     perfc_add(kr->pc, PERFC_RA_CNCOMP_RBYTES, iov.iov_len);
+#endif
```

Comment:
Open a ticket for this, there's needs to be some proper accounting somewhere to prevent this from happening.